crates/rooch/src/commands/move_cli/commands/unit_test.rs

```diff
@@ -36,6 +43,45 @@ impl Test {
             .into_iter()
             .map(|(key, value)| (key, AccountAddress::from_hex_literal(&value).unwrap()))
             .collect();
+
+        let root_path = path.clone().unwrap_or_else(|| PathBuf::from("."));
+
+        build_config.dev_mode = true;
+
+        let resolution_graph = build_config
+            .clone()
+            .resolution_graph_for_package(&root_path, &mut Vec::new())
+            .expect("resolve package dep failed");
+
+        let mut additional_named_address = BTreeMap::new();
+        let _: Vec<_> = resolution_graph
+            .extract_named_address_mapping()
+            .map(|(name, addr)| {
+                (additional_named_address.insert(
+                    name.to_string(),
+                    NumericalAddress::new(addr.into_bytes(), NumberFormat::Hex).into_inner(),
+                ),)
+            })
+            .collect();
+
+        let global_env = build_model_internal(&root_path, additional_named_address, None)?;
+
+        let _ = run_extended_checks(&global_env);
+
+        if global_env.diag_count(Severity::Warning) > 0 {
+            let mut buffer = Buffer::ansi();
+            global_env.report_diag(&mut buffer, Severity::Warning);
+            let buffer_output = String::from_utf8_lossy(buffer.as_slice()).to_string();
+            let re = Regex::new("(/.*)(.move:[0-9]+:[0-9]+)").unwrap();
+            let ret = re
+                .replace(buffer_output.as_str(), "/tmp/tempfile$2".to_string())
+                .to_string();
```

Comment:
do not need to replace output file names in unit tests output.

Reply:
That's correct, here it has been changed to not replace the content.